src/panels/MissionsPanel/MissionTimeButtons.tsx

```diff
@@ -0,0 +1,64 @@
+import { Button, Group } from '@mantine/core';
+
+import { GoToEnd, GoToStart, TimerIcon } from '@/icons/icons';
+import { DisplayType } from '@/types/enums';
+
+import { useJumpToTime } from './hooks';
+import { DisplayedPhase } from './MissionContent';
+
+interface MissionTimeButtonsProps {
+  currentPhase: DisplayedPhase;
+}
+
+export function MissionTimeButtons({ currentPhase }: MissionTimeButtonsProps) {
+  const jumpToTime = useJumpToTime();
+  const isMissionOverview = currentPhase.type === DisplayType.Overview;
+
+  function jumpToEndOfPhase() {
+    if (currentPhase.type === DisplayType.Phase || isMissionOverview) {
+      jumpToTime(currentPhase.data.timerange.end);
```

Comment:
This should set the time to _just before_ the phase ends, in my opinion. Now there is some weirdness going on if if one has the "current phase" toggle selected and click this button. 

It will either just be empty, or even result in some weird flickering behavior

https://github.com/user-attachments/assets/685ff590-aeae-4ecc-80e7-7823e2da3714

Reply:
I can't reproduce this 🤔